platform/jvm/replay/src/main/kotlin/io/bitdrift/capture/replay/internal/ScreenshotCaptureEngine.kt

```diff
@@ -71,16 +71,17 @@ internal class ScreenshotCaptureEngine(
                 .ofWindow(topView)
                 .build()
         PixelCopy.request(screenshotRequest, executor) { screenshotResult ->
-            val resultBitmap = screenshotResult.bitmap
```

Comment:
there's a `validateStatus()` inside the getter that throws an `IllegalStateException` when the `status != SUCCESS`